libc-bottom-half/crt/crt1-command.c

```diff
@@ -15,6 +21,8 @@ void _start(void) {
     }
     started = 1;
 
+	__wasi_init_tp();
```

Comment:
When you resolve the merge conflict, be sure to put this in the `_REENTRANT` path :-).